### DISCUSSION TITLE
Is it possible to add anchor linking (#scroll-here) to relation managers?

### DISCUSSION THREAD

**Discussion by joshembling** [filamentphp/filament#6840] [General]
As per title, is it possible to add an id to a relation manager, so you can externally link to that part of the page. E.g. `/resource/123/edit?activeRelationManager=0#my-relation-manager`

I suppose similarly to what `getTableQueryStringIdentifier()` is doing, but for a global ID. 

Thanks in advance!

**Comment by pboivin** [comment]
I think this can be done if you customize your page's Blade view :
https://github.com/filamentphp/filament/blob/2.x/packages/admin/resources/views/resources/pages/edit-record.blade.php#L35
